go/zeusmaster/zeusmaster_test.go

```diff
@@ -0,0 +1,265 @@
+package zeusmaster_test
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"path"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/burke/zeus/go/config"
+	slog "github.com/burke/zeus/go/shinylog"
+	"github.com/burke/zeus/go/unixsocket"
+	"github.com/burke/zeus/go/zeusmaster"
+)
+
+var testFiles = map[string]string{
+	"zeus.json": `
```

Comment:
Yep, gross either way. Having this code live next to the code that generates/uses it is not too bad anyway - let's stick to this.